### PR TITLE
cart count

### PR DESCRIPTION
**Pull request by YuraBuchak**

### CODE REVIEW

File: src/pages/product-page/index.tsx [hunk 1]
```diff
@@ -10,10 +10,16 @@ import IconArrowRight from '../../icons/arrowright.svg?react'
...
+import { useDispatch, useSelector } from 'react-redux'
+import { AppDispatch, RootState } from '../../redux/store'
+import Notiflix, { Notify } from 'notiflix'
 import { formatWeight } from '../../helpers/formatWeightToString'
+import { fetchBasketItemsThunk } from '../../redux/basket-item-count/operations'
+import { updateCount } from '../../redux/basket-item-count/slice'
+
+Notiflix.Notify.init({
+  position: 'center-top'
+})
```
Comment (pqoqubbw) [inline src/pages/product-page/index.tsx:22]: why do you init it in this component?

File: src/redux/basket-item-count/selectors.tsx [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const selectBasketItemCount = (state) =>
```
Comment (pqoqubbw) [inline src/redux/basket-item-count/selectors.tsx]: where do you use this? we decided to remove selectors at all

File: src/redux/basket-item-count/slice.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { fetchBasketItemsThunk } from './operations'
+
+const initialState: { basketItemCount: number } = {
```
Comment (pqoqubbw) [inline src/redux/basket-item-count/slice.tsx]: this should be as a `type`, not just inline type for variable

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/pages/product-page/index.tsx (hunk 1 above)
Comment (YuraBuchak) [reply src/pages/product-page/index.tsx:22]: cause these are temporary notifications and default settings interfere with testing

File: src/redux/basket-item-count/selectors.tsx (hunk 2 above)
Comment (YuraBuchak) [reply src/redux/basket-item-count/selectors.tsx]: removed

File: src/redux/basket-item-count/slice.tsx (hunk 3 above)
Comment (YuraBuchak) [reply src/redux/basket-item-count/slice.tsx]: done

File: src/components/modal-product-limits/ModalProductLimits.module.scss [hunk 4]
```diff
@@ -1,23 +1,28 @@
 @import '../../styles/base/base.scss';
 
+.popover {
+  position: relative;
+  width: 400px;
+  overflow-y: visible !important;
+  overflow-x: visible !important;
```
Comment (pqoqubbw) [inline src/components/modal-product-limits/ModalProductLimits.module.scss:7]:
```suggestion
  overflow: visible !important;
```

**Review by pqoqubbw — APPROVED** [review]

**Pull request merged**